src/layout/section/cardsGridSection/CardsGridSection.style.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components'
+
+import { color, componentSizes, responsiveBreakpoints, space } from '../../../_utils/branding'
+
+const wrapperHorizontalPadding = space.xl
+const cardsPerRowSmall = 2
+const cardsPerRowLarge = 3
+
+export const StyledCardsGridSection = styled.div`
+  & .kirk-cardsGridSection-title {
+    margin: ${space.l} 0 ${space.m};
+  }
+
+  & .kirk-cardsGridSection-button {
+    text-align: center;
+  }
+
+  & .kirk-cardsGridSection {
+    margin: auto;
+    display: flex;
+    justify-content: start;
+    padding: ${space.l} 0;
+    box-sizing: border-box;
+    overflow: auto; /* Make Cards scrollable horizontally */
+    -ms-overflow-style: none; /* Remove scrollbar visually IE 10+ */
+    scrollbar-width: none; /* Remove scrollbar visually Firefox */
+    scroll-snap-type: x mandatory;
+    scroll-padding: ${space.xl};
+  }
+
+  & .kirk-cardsGridSection .kirk-card {
+    scroll-snap-align: start;
+    overflow: hidden;
+  }
+
+  /* Remove scrollbar webkit */
+  & .kirk-cardsGridSection::-webkit-scrollbar {
+    display: none;
+  }
+
+  & .kirk-card {
+    flex: 1;
+    margin: 0 ${space.l} 0 0;
+    background-color: ${color.white};
+    position: relative;
+  }
+
+  & .kirk-card:last-child {
+    margin-right: 0;
+  }
+
+  /* The browser does not add right padding at the end of the scroll, CSS solution:
+  * https://blog.alexandergottlieb.com/overflow-scroll-and-the-right-padding-problem-a-css-only-solution-6d442915b3f4
+  */
+  & .kirk-card:last-child::after {
+    content: '';
+    display: block;
+    position: absolute;
+    right: -${wrapperHorizontalPadding};
+    width: ${wrapperHorizontalPadding};
+    height: 1px;
+  }
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
```

Comment:
nit: By declaring breakpoints by a component we easily identify its "global" behavior 

for exemple: 

```suggestion
  & .kirk-cardsGridSection {
    // Mobile first behavior
    
     @media (${responsiveBreakpoints.isMediaLarge}) {
        // Large behavior
     }
}
  
```

Reply:
I'll remove the media rule for `small` and make it the default, and keep only the rule for media `large`.
So all the behavior for `large` is referenced at the same level.